Makefile

```diff
@@ -140,14 +139,9 @@ fmt: ; $(info  running gofmt...) @ ## Run gofmt on all source files
 	 done ; exit $$ret
 
 # Dependency management
-
-glide.lock: glide.yaml | $(BASE) ; $(info  updating dependencies...)
-	$Q cd $(BASE) && $(GLIDE) update -v
-	@touch $@
-vendor: glide.lock | $(BASE) ; $(info  retrieving dependencies...)
-	$Q cd $(BASE) && $(GLIDE) --quiet install -v
-	@ln -nsf . vendor/src
-	@touch $@
+.PHONY: deps-update
+deps-update: $(info  updating dependencies...)
```

Comment:
Missing comma after `deps-update: `. It makes `updating dependencies` info always be printed when `make` is executed even if it's not updating dependency.

Reply:
This for the comment!

I fix it :)